docs/src/nodes/detailed-hydropower.md

```diff
@@ -0,0 +1,780 @@
+# [Detailed hydropower](@id nodes-det_hydro_power)
```

Comment:
The current layout requires that each `HydroReservoir` has a link going into it. Otherwise, the variable `flow_in` is unconstrained, resulting in power generation from the `HydroReservoir` only being limited by the connected `HydroGenerator`.

A first solution to this problem is to specify that the a hydro power system requires input couplings to all `HydroReservoir`s. For `HydroReservoir` not coupled to a `HydroPump`, this can be achieved through a `RefSink` with the water resource as output and a capacity of 0.

Reply:
Solved in https://github.com/EnergyModelsX/EnergyModelsRenewableProducers.jl/pull/35/commits/3427df3efaa402306842698a9b27c76f9433d177